### PR TITLE
loyalty: allow merchants to enable loyalty feature only if it is enabled in the community

### PR DESCRIPTION
**Pull request by lucasvanhalst**
Closes https://github.com/our-city-app/planning/issues/14

added the feature and removed some unused stuff

### CODE REVIEW

File: src/static/js/solutions/sln-functionalities.js [hunk 1]
```diff
@@ -235,11 +228,7 @@ $(function() {
                 return true;
             } else if (button.hasClass('btn-danger')) {
                 return false;
-            } else if (button.attr('action') === 'on') {
-                return false;
-            } else {
-                return true;
```
Comment (RubenMattan) [inline src/static/js/solutions/sln-functionalities.js:241]: else is not used?

Comment (lucasvanhalst) [reply src/static/js/solutions/sln-functionalities.js:241]: the functionality is the same it's just been shortened

**Comment by RubenMattan** [comment]
- [ ] if they already have the module active lets keep showing it in the dashboard
- [ ] get_allowed_modules & POSSIBLE_MODULES (can be others) needs to be updated

**Comment by RubenMattan** [comment]
Let's create a migration also for every service that has this module but no transactions and remove the module.
For now let's keep it for services that have transactions. we should create a list of those and let Steven decide what action to take for those

**Pull request merged**